instrumentation/instaecho/README.md

```diff
@@ -0,0 +1,40 @@
+Instana instrumentation for echo framework
+=============================================
+
+This module contains middleware to instrument HTTP services written with [github.com/labstack/echo](https://github.com/labstack/echo).
+
+[![GoDoc](https://img.shields.io/static/v1?label=godoc&message=reference&color=blue)][godoc]
+
+
+Installation
+------------
+
+To add the module to your `go.mod` file run the following command in your project directory:
+
+```bash
+$ go get github.com/instana/go-sensor/instrumentation/instaecho
+```
+
+Usage
+-----
+
+```go
+// init Echo
+e := echo.New()
+
+// create a sensor
+sensor := instana.NewSensor("echo-sensor")
+
+// add middleware to the Echo's handlers
```

Comment:
We should emphasize that middleware should be added before declaring any handlers:
> Middleware registered using `Echo#Use()` is only executed for paths which are registered after `Echo#Use()` has been called.

[Source](https://echo.labstack.com/middleware/)

Reply:
Makes sense! Good catch! 👍